src/test/rgw/test_store_bench.cc

```diff
@@ -0,0 +1,281 @@
+#include "common/async/yield_context.h"
+#include "common/ceph_argparse.h"
+#include "common/ceph_context.h"
+#include "common/ceph_time.h"
+#include "common/code_environment.h"
+#include "common/common_init.h"
+#include "common/dout.h"
+#include "include/buffer_fwd.h"
+#include "include/msgr.h"
+#include "include/rados/librados.hpp"
+#include "rgw_basic_types.h"
+#include "rgw_common.h"
+#include "rgw_op.h"
+#include "rgw_sal.h"
+#include "rgw_sal_rados.h"
+#include "rgw_tools.h"
+#include "test/librados/test_shared.h"
+#include "gtest/gtest.h"
+#include <bits/stdint-intn.h>
+#include <bits/stdint-uintn.h>
+#include <boost/optional/optional.hpp>
+#include <boost/variant.hpp>
+#include <chrono>
+#include <climits>
+#include <cstdint>
+#include <map>
+#include <memory>
+#include <string>
+
+#include "test/librados/test_cxx.h"
+
+#define CONF Config::instance()
+
+class Config {
+public:
+	DoutPrefixProvider *dpp;
+	CephContext *cct;
+	rgw::sal::Store *store;
+	RGWRados *rados;
```

Comment:
This appears to be unused.

Reply:
I'm using the `Config::instance()` with the `CONF` macro everytime I want to get these. The macro is there so I don't have to type `Config::instance()` everytime tbh